exchanges/account/account.go

```diff
@@ -342,3 +358,16 @@ func (b *ProtectedBalance) GetFree() float64 {
 	defer b.m.Unlock()
 	return b.free
 }
+
+func (b *ProtectedBalance) clean() {
```

Comment:
🐒  `clear`, `empty`, `reset`, or `zero` (as a verb)
```suggestion
func (b *ProtectedBalance) reset() {
```